apps/passport-client/src/broadcastChannel.ts

```diff
@@ -0,0 +1,46 @@
+// Note: refactoring to @pcd/passport-interface in the future, especially if
+// cross-browser interaction starts to exist outside of passport-client
+import { Action } from "./dispatch";
+
+const CHANNEL_NAME = "zupass_broadcast_channel";
+// The event message that prompts other tabs to refresh their local state
+const PASSWORD_CHANGE_ON_OTHER_TAB_MESSAGE = "passwod_change_on_other_tab";
```

Comment:
Assuming a BroadcastChannel can accept objects not strings, it might make sense to prepare for the possibility of messages with payload by defining the message as a struct like `{message: "password_change_on_other_tab" /*payload here in future*/}`